src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -19,6 +19,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RunCommand Condition="'$(RunCommand)' == ''">dotnet</RunCommand>
 
     <WasmAppHostDir>$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'WasmAppHost'))</WasmAppHostDir>
+    <WasmBootConfigFileName Condition="'$(WasmBootConfigFileName)'==''">blazor.boot.json</WasmBootConfigFileName>
+    <WasmPublishBootConfigFileName Condition="'$(WasmPublishBootConfigFileName)'==''">publish.$(WasmBootConfigFileName)</WasmPublishBootConfigFileName>
```

Comment:
Do you plan to set this property or is the computed value fine for you? I would prefer to have it as private property / implementation detail and thus prefix it with underscore.

Reply:
I don't, only through the `WasmBootConfigFileName`  file.